userguide/text_classifier/README.md

```diff
@@ -25,35 +25,53 @@ sentiment for each aspect. For example, when studying reviews about
 mobile phones you may be interested in how people feel about aspects
 such as battery life, screen resolution, size, etc.
 
+#### Loading Data
+
+The [Kaggle SMS Spam Collection Dataset](https://www.kaggle.com/uciml/sms-spam-collection-dataset) provides more than five thousand labeled text messages.[<sup>1</sup>](../datasets.md) After downloading and decompressing the dataset, move the `spam.csv` file to your project directory.
```

Comment:
Please use this link instead: 
https://archive.ics.uci.edu/ml/datasets/SMS+Spam+Collection

And just refer to it as the "SMS Spam Collection Dataset"